covasim/webapp/index.html

```diff
@@ -36,292 +36,300 @@
+    <template v-else>
+
+      <div class="d-flex flex-wrap">
+        <div class="w-50" v-for="x in result.graphs">
+          <plotly-chart :graph="x" :key="x.id"/>
         </div>
-      </div>
 
-      <fieldset>
-        <legend>Results</legend>
-        <!--
-        <template v-if="running">
-          <p><i>Please wait, running model...</i></p>
-            <b-spinner />
-        </template>
-        -->
-        <template v-if="result.graphs.length === 0">
-          <p><i>Click "Run" to display graphs</i></p>
-        </template>
-        <template v-else>
-
-          <div class="d-flex flex-wrap">
-            <div class="w-50" v-for="x in result.graphs">
-              <plotly-chart :graph="x" :key="x.id"/>
-            </div>
+        <!-- <b>After {{ result.summary.days }} days: {{ result.summary.cases }} cases, {{ result.summary.deaths }} deaths</b> -->
 
-            <!-- <b>After {{ result.summary.days }} days: {{ result.summary.cases }} cases, {{ result.summary.deaths }} deaths</b> -->
-
-            <div class="text-center">
-              <button id="download_xlsx" class="btn btn-primary blueish" type="button" @click="downloadExcel">
-                <span class="sr-only">Download</span> XLSX <span class="ti-download"/>
-              </button>
-              <button id="download_json" class="btn btn-primary blueish" type="button" @click="downloadJson">
-                <span class="sr-only">Download</span> JSON <span class="ti-download"/>
-              </button>
-              <b-tooltip target="download_xlsx">
-                Download results as an XLSX file
-              </b-tooltip>
-              <b-tooltip target="download_json">
-                Download results as a JSON file
-              </b-tooltip>
-            </div>
+        <div class="text-center">
+          <button id="download_xlsx" class="btn btn-primary blueish" type="button" @click="downloadExcel">
+            <span class="sr-only">Download</span> XLSX <span class="ti-download"/>
+          </button>
+          <button id="download_json" class="btn btn-primary blueish" type="button" @click="downloadJson">
+            <span class="sr-only">Download</span> JSON <span class="ti-download"/>
+          </button>
+          <b-tooltip target="download_xlsx">
+            Download results as an XLSX file
+          </b-tooltip>
+          <b-tooltip target="download_json">
+            Download results as a JSON file
+          </b-tooltip>
+        </div>
 
-          </div>
+      </div>
 
-        </template>
+    </template>
 
-      </fieldset>
-    </div>
+  </fieldset>
+  </div>
   </div>
 
   <footer class="row no-gutter">
     <b-navbar type="dark" variant="dark">
-      <span class="navbar-text text-white">
-        &copy; {{ app.copyright_year }} <a :href="app.org_url" target="_blank">Institute for Disease Modeling</a>
-      </span>
+      <a :href="app.org_url" target="_blank">
+        <img src="assets/idm-logo.png" height="30px"/>
```

Comment:
Don't set height directly on an image. Use bootstrap responsive image classes.

Reply:
I tried `<img src="assets/idm-logo.png" class="img-responsive"/>` but that didn't work...not sure what else needs to change

Reply:
That's because we're using Bootstrap 4 (this is why I pinned the versions, it makes it easier to look up the docs and such!). The class is 'img-fluid'. You should also add in cova_app.css a max-height for the footer images of 2em and make sure the text aligns properly.

Reply:
I agree this would be better but I won't have time to fix straight away -- I'll make an issue